### PR TITLE
Adjust prewhere_with_row_level_filter performance test

### PR DESCRIPTION
**Pull request by traceon**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: docker/test/performance-comparison/config/config.d/user_files.xml [hunk 1]
```diff
@@ -4,4 +4,7 @@
 
     <!-- Path to configuration file with users, access rights, profiles of settings, quotas. -->
     <users_config>users.xml</users_config>
-</yandex>
\ No newline at end of file
+
+    <!-- Path to directory where users created by SQL commands are stored. -->
+    <access_control_path>access/</access_control_path>
```
Comment (KochetovNicolai) [inline docker/test/performance-comparison/config/config.d/user_files.xml:9]: Hm, does relative path work?

**Review by KochetovNicolai — APPROVED** [review]

File: docker/test/performance-comparison/config/config.d/user_files.xml (hunk 1 above)
Comment (traceon) [reply docker/test/performance-comparison/config/config.d/user_files.xml:9]: Should work, but overall, this PR is still in "blind fix" state. I am waiting for CI results...

File: tests/performance/prewhere_with_row_level_filter.xml [hunk 2]
```diff
@@ -1,16 +1,15 @@
 <test>
     <create_query>DROP TABLE IF EXISTS test_prl;</create_query>
     <create_query>CREATE TABLE test_prl (n UInt64) ENGINE MergeTree ORDER BY n;</create_query>
-    <!--<create_query>GRANT CREATE ROW POLICY ON *.* TO CURRENT_USER</create_query>-->
     <create_query>CREATE ROW POLICY OR REPLACE test_prl_policy ON test_prl AS PERMISSIVE FOR SELECT USING n % 7 TO ALL;</create_query>
 
-    <fill_query>INSERT INTO test_prl SELECT number FROM numbers(500000000);</fill_query>
+    <fill_query>INSERT INTO test_prl SELECT number FROM numbers(100000000);</fill_query>
```
Comment (KochetovNicolai) [inline tests/performance/prewhere_with_row_level_filter.xml:6]: Maybe we need to enable parallel insert select

Comment (traceon) [reply tests/performance/prewhere_with_row_level_filter.xml:6]: Change to something like this, you mean?
```sql
INSERT INTO test_prl SELECT number FROM numbers(100000000) SETTINGS max_insert_threads = 10;
```

Comment (KochetovNicolai) [reply tests/performance/prewhere_with_row_level_filter.xml:6]: yep

Comment (traceon) [reply tests/performance/prewhere_with_row_level_filter.xml]: Pushed the change.

**Pull request merged**